test/HealthChecks.SqlServer.Tests/DependencyInjection/RegistrationTests.cs

```diff
@@ -42,7 +42,7 @@ void beforeOpen(SqlConnection connection)
         var registration = options.Value.Registrations.First();
         var check = registration.Factory(serviceProvider);
 
-        Record.ExceptionAsync(() => check.CheckHealthAsync(new HealthCheckContext())).GetAwaiter().GetResult();
+        await Record.ExceptionAsync(() => check.CheckHealthAsync(new HealthCheckContext()));

```

Comment:
xUnit1031: Test methods should not use blocking task operations, as they can cause deadlocks.